src/controllers/userTransform.controller.ts

```diff
@@ -0,0 +1,80 @@
+import { Context } from "koa";
+import MiscService from "../services/misc.service";
+import { ProcessorRequest, UserTransfromServiceResponse } from "../types/index";
+import { compileUserLibrary } from "../util/ivmFactory";
+import UserTransformService from "../services/userTransform/userTransform.service";
+import logger from "../logger";
+import { setupUserTransformHandler } from "../util/customTransformer";
+import ControllerUtility from "./util";
+
+export default class UserTransformController {
+  public static async userTransform(ctx: Context) {
```

Comment:
But without dir segregation during import it would be confusing to import and find

Reply:
I meant to fix the method name `userTransform` as the class name already establishes the context.

Reply:
![Screenshot 2023-01-02 at 11 13 20 AM](https://user-images.githubusercontent.com/25389167/210198200-6969bed0-e2c9-4f51-bb55-565e9342bf21.png)
Currently this is the structure I kept if I remove `user` it might become confusing while user is looking at just file name

Reply:
I'm not asking to remove "user" from the file name but from the method name inside it, `userTransform` to `transform`. However, looks like you've already addressed that.